docs/articles/documentation/extending-testcafe/custom-reporter/index.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Custom Reporter Plugin
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/
+---
+# Custom Reporter Plugin
+
+TestCafe has a number of [built-in reporters](/testcafe/documentation/using-testcafe/common-concepts/reporters/) to generate test reports in different formats.
+You can also create a **custom reporter** that will output information in your own format and style.
+For this purpose, you can use the [TestCafe reporter generator](https://github.com/DevExpress/generator-testcafe-reporter).
+The generator will scaffold out a reporter plugin, so that you only need to write a few lines of code.
+
+For more information, refer the following topics.
+
+* [Creating Reporter Plugin](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/creating-reporter-plugin/)
+* [Helper Methods](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/helper-methods)
```

Comment:
Another integral solution is to remove *Helper Methods* from *Custom Reporter Plugin* AND  the side menu as well. This way, we'll have it as a kind of hidden topic.
And I still have reservations...